fabric-api-base/src/testmodClient/java/net/fabricmc/fabric/test/base/client/TestDedicatedServer.java

```diff
@@ -51,25 +49,14 @@ public String getConnectionAddress() {
 	}
 
 	public void runCommand(String command) {
-		submitAndWait(server -> {
-			server.enqueueCommand(command, server.getCommandSource());
-			return null;
-		});
+		ThreadingImpl.runOnServer(() -> server.getCommandManager().executeWithPrefix(server.getCommandSource(), command));
 	}
 
 	private void run() {
 		setupServer();
 		Main.main(new String[]{});
 	}
 
-	private <T> CompletableFuture<T> submit(Function<MinecraftDedicatedServer, T> function) {
-		return server.submit(() -> function.apply(server));
-	}
-
-	private <T> T submitAndWait(Function<MinecraftDedicatedServer, T> function) {
-		return submit(function).join();
-	}
-
```

Comment:
Why are there no `runOnServer()` / `computeOnServer()` helpers here? I mean, I can't say that I've used the old `TestDedicatedServer.submitAndWait()` in actual test code, but this seems like something that more server-heavy mods would find useful in their tests. Obviously one can run `ThreadingImpl.runOnServer()` directly instead, as you've done above, but it feels inconsistent to have these helpers on one side and not the other.

Reply:
They were private and unused so I deleted them, the equivalent is in `ThreadingImpl`. We can think about what helpers are useful to expose in a future PR

Reply:
OK.